backend/src/main.ts

```diff
@@ -19,9 +20,9 @@ async function bootstrap() {
   // Swagger
   if (process.env.NODE_ENV === 'development') {
     const config = new DocumentBuilder()
-      .setTitle('BeyondLife Api')
-      .setDescription('The BeyondLife Api')
-      .setVersion('0.1')
+      .setTitle('Siebtesleben Api')
```

Comment:
You can leave this in...but it is not determined yet if the repository will be renamed